src/modules/info-panel/info-panel.tag.html

```diff
@@ -33,8 +33,10 @@
     <script>
         var tag = this;
         tag.activeSection = undefined;
+        tag.isMaximized = false;
         opts.sections = {};
         tag.onSelectSection = onSelectSection;
+        tag.onMaximize = onMaximize;
```

Comment:
I'd change `onMaximize` to `toggleMaximize` as it toggles a property rather than being some event callback.